---
name: 'Understanding the key role of accessory genes in AMR phenotype through interpretable machine learning techniques.'
speakers:
	- Lucy Dillon
categories:
	-
---
Background: Antimicrobial resistance (AMR) genes are found to be ubiquitous within the microbiome, even when antimicrobial usage is absent. To identify AMR phenotype, the most common method is to use a laboratory-based assay. Yet, when dealing with samples from the microbiome, many species are difficult to culture within the laboratory. The vast quantity of strains would be time-consuming to culture. To avoid this, a computational approach may be a more favourable choice. AMR gene finder tools are efficient at determining the AMR genotype. Despite this, how AMR genotype relates to AMR phenotype is still an open question.  Methods: To evaluate the relationship between AMR phenotype and AMR genotype 16,950 genomes from BV-BRC which had corresponding MIC values were analysed. Using Wekaâ€™s J48 decision tree model the relationship between AMR phenotype and AMR genotype was analysed. The role of accessory genes in relation to AMR phenotype was analysed in the same way.   Results: The J48 models could predict AMR phenotype accurately using AMR genes and accessory genes, the average accuracy was 91.7% and 92.2% respectively. The results found that gene co-occurrence, presence and absence of genes are key factors to analyse when identifying AMR phenotype from genomic data. These factors are not evaluated by commonly used AMR gene finder tools, which could miss vital information to determine the correct phenotype. Our results highlight why we should continue to research the relationship between AMR phenotype and genomic data.
